packages/cli-main/src/cli/commands/auth/logout.ts

```diff
@@ -1,12 +1,11 @@
 import Command from '@shopify/cli-kit/node/base-command'
-import {output, session, store} from '@shopify/cli-kit'
+import {output, session} from '@shopify/cli-kit'
 
 export default class Logout extends Command {
   static description = 'Logout from Shopify'
 
   async run(): Promise<void> {
     await session.logout()
-    await store.removeSession()
```

Comment:
this was removed because `session.logout` already calls `store.removeSession` if needed